content/tutorials/provision-dashboards-and-data-sources.md

```diff
@@ -60,33 +63,33 @@ provisioning/
     <yaml files>
 ```
 
-Next, let's look at how to provision a data source.
+Next, we'll look at how to provision a data source.
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Provision a data source" %}}
 
-Each data source config file contains a _manifest_ that specifies the desired state of a set of data sources.
+Each data source configuration file contains a _manifest_ that specifies the desired state of a set of data sources. ((I'm a little lost. Is this specific to provisioning or is it in general? I infer from the next paragraph that it is probably provisioning-only, but you should probably clarify that up front.))
 
-At start-up, Grafana loads the configuration files and provisions the data sources listed in the manifests.
+At startup, Grafana loads the configuration files and provisions the data sources listed in the manifests.
 
 Let's configure a [TestData DB](https://grafana.com/docs/grafana/latest/features/datasources/testdata/) data source that you can use for your dashboards.
 
 #### Create a data source manifest
 
-- In the `provisioning/datasources/` directory, create a file called `default.yaml` with the following content:
+1. In the `provisioning/datasources/` directory, create a file called `default.yaml` with the following content:
```

Comment:
Unfortunately, nested code blocks doesn't work for the learning platform. Not sure if it's Hugo-specific, or a bug in our code. This will reset the numbering after the code block.

Reply:
Bugger. It worked in my preview.

Reply:
Will it work if we hard code the numbers?

Reply:
Oh, maybe it's fixed then! Let's leave it like this then and check it out in staging.